yt_dlp/extractor/youtube.py

```diff
@@ -3554,11 +3692,24 @@ def _real_extract(self, url):
         microformats = traverse_obj(
             player_responses, (..., 'microformat', 'playerMicroformatRenderer'),
             expected_type=dict, default=[])
-        video_title = (
-            get_first(video_details, 'title')
-            or self._get_text(microformats, (..., 'title'))
-            or search_meta(['og:title', 'twitter:title', 'title']))
-        video_description = get_first(video_details, 'shortDescription')
+
+        translated_title = self._get_text(microformats, (..., 'title'))
+        video_title = (self._preferred_lang and translated_title
+                       or get_first(video_details, 'title')  # primary
+                       or translated_title
+                       or search_meta(['og:title', 'twitter:title', 'title']))
+        translated_description = self._get_text(microformats, (..., 'description'))
+        original_description = get_first(video_details, 'shortDescription')
+        video_description = (
+            self._preferred_lang and translated_description
+            or (
+                # If original description is blank, it will be an empty string.
+                # Do not prefer translated description in this case.
+                (original_description or None)
+                if original_description is not None
+                else (translated_description or None)
```

Comment:
If original_description is blank, wouldn't translated_description also be? If so, this an be simplified to `original_description or translated_description`.

Even if not, why the "or None"?

Reply:
```suggestion
                # If original description is blank, it will be an empty string.
                # Do not prefer translated description in this case.
                translated_description if original_description is None else original_description
```
should be enough imo